source/server/BUILD

```diff
@@ -196,6 +196,8 @@ envoy_cc_library(
         "//conditions:default": [],
     }),
     external_deps = ["tclap"],
+    # TCLAP command line parser needs this to support int64_t/uint64_t in several build environments.
+    copts = ["-DHAVE_LONG_LONG"],
```

Comment:
@zuercher as the original person that brought in the copt. This is only needed in code that uses tclap. It can't be pushed all the way to the tclap.BUILD file because at that moment we are not using uint64_t/int64_t. Do you think this needs a special bazel rule so that people include and it automatically also puts in the copt in the rule? I think that might be overkill given this is just two places.

Reply:
Yeah I think this is good.

Reply:
Ok by me. The compiler error when it's not defined is not super helpful, but hopefully any future deps on tclap will just copy/paste this flag.